### PR TITLE
Add notes for obsolete argument to navigator.registerProtocolHandler

### PR DESCRIPTION
**Pull request by Rumyra**
A checklist to help your pull request get merged faster:
- [x] Summarize your changes

Adding notes to browsers that have not changed the implementation as per spec, to remove the title arg

- [x] Data: link to resources that verify support information (such as browser's docs, changelogs, source control, bug trackers, and tests)

Firefox has as per https://bugzilla.mozilla.org/show_bug.cgi?id=1631464 spec is here https://html.spec.whatwg.org/multipage/system-state.html#custom-handlers

- [x] Data: if you tested something, describe how you tested with details like browser and version

Tested in FF 84, Chrome 86 and Opera 72 on OSX. Third param is needed for Chrome and Opera, made a judgement call it's the same for Edge as well

- [x] Review the results of the linter and fix problems reported (If you need help, please ask in a comment!)

All good

- [x] Link to related issues or pull requests, if any

https://github.com/mdn/sprints/issues/3899

### CODE REVIEW

**Comment by chrisdavidmills** [comment]
Can I get a request of this one please @ddbeck ? This is related to the Fx 84 launch (Dec 15). Thanks!

File: api/Navigator.json [hunk 1]
```diff
@@ -1832,15 +1832,19 @@
               "version_added": "13",
               "notes": [
                 "Allowed schemes include <code>mailto</code>, <code>mms</code>, <code>nntp</code>, <code>rtsp</code>, and <code>webcal</code>. Custom protocols must be prefixed with <code>web+</code>.",
-                "From Chrome 77, the URL parameter only accepts <code>http</code> or <code>https</code> URLs."
+                "From Chrome 77, the URL parameter only accepts <code>http</code> or <code>https</code> URLs.",
+                "The third <code>title</code> argument has not been removed and so will still need to be specified when using this method"
```
Comment (ddbeck) [inline api/Navigator.json:1836]: suggestion: revise the note to make it a bit more direct about what Web developers need to do with this information

```suggestion
                "The obsolete <code>title</code> argument is required."
```

File: api/Navigator.json [hunk 2]
```diff
@@ -1832,15 +1832,19 @@
...
             },
             "chrome_android": {
               "version_added": false
             },
             "edge": {
               "version_added": "79",
-              "notes": "Allowed schemes include <code>mailto</code>, <code>mms</code>, <code>nntp</code>, <code>rtsp</code>, and <code>webcal</code>. Custom protocols must be prefixed with <code>web+</code>."
+              "notes": [
+                "Allowed schemes include <code>mailto</code>, <code>mms</code>, <code>nntp</code>, <code>rtsp</code>, and <code>webcal</code>. Custom protocols must be prefixed with <code>web+</code>.",
+                "The third <code>title</code> argument has not been removed and so will still need to be specified when using this method"
```
Comment (ddbeck) [inline api/Navigator.json:1846]:
```suggestion
                "The obsolete <code>title</code> argument is required."
```

File: api/Navigator.json [hunk 3]
```diff
@@ -1852,7 +1856,8 @@
               "version_added": false
             },
             "opera": {
-              "version_added": "11.6"
+              "version_added": "11.6",
+              "notes": "The third <code>title</code> argument has not been removed and so will still need to be specified when using this method"
```
Comment (ddbeck) [inline api/Navigator.json:1860]:
```suggestion
              "notes": "The obsolete <code>title</code> argument is required."
```

**Review by ddbeck — CHANGES_REQUESTED** [review]
An editing suggestion in line comments. Thank you!

**Review by ddbeck — APPROVED** [review]
Looks good. Thank you! 🎉

**Pull request merged**